stories/optionButton/optionButtonExample.js

```diff
@@ -37,15 +36,6 @@ class OptionButtonExample extends React.Component {
           type="primary"
           icon={<PlusIcon />}
         />
-        <OptionButton
```

Comment:
was thi removed in other PR?

Reply:
sry  I refer to the `EqualsIcon`

Reply:
Just aesthetic. The new `EqualsIcon` is now wrapped in a circle, and so it looks like this:

<img width="462" alt="Screen Shot 2020-01-29 at 1 08 25 PM" src="https://user-images.githubusercontent.com/25893608/73397511-8ad68c00-4298-11ea-8d14-b41ce6dfcd1d.png">

It's on my radar to check how it's used in the main repos, but here it was just being used as an example, and it it was looking strange next to the plus and minus icons.

Reply:
yeah that looks weird.